.github/workflows/ci.yml

```diff
@@ -103,8 +105,13 @@ jobs:
       - name: Change admin password to make it easier for Cypress tests
         run: ddev drush upwd "$CYPRESS_ADMIN_USERNAME" "$CYPRESS_ADMIN_PASSWORD"
 
-      - name: Cypress Test
-        run: ddev cypress-run
+      # - name: Cypress Test
+      #   run: ddev cypress-run
+
+      - name: Cypress Test with Percy Integration
+        run: |
+          ddev exec --service cypress "git config --global --add safe.directory /e2e"
```

Comment:
Is this needed? Why would percy need to use git in that directory?

Reply:
Percy tries to identify some Git information, e.g., the current branch, commit hash, PR number, and author etc so it can use the same info in percy.io. If nothing is found, it just says unknown-branch.

Earlier, I tried to set these details via environment variables. Please see [this commit](https://github.com/contrib-tracker/backend/pull/467/commits/bd8efd6fcfeb08c496ba7b400950aba53d3dd1c3).

https://www.browserstack.com/docs/percy/get-started/set-env-var

But then I realized that Git is not added to the safe directory, which is why Percy is not able to identify the Git details.